lib/schema.js

```diff
@@ -4,15 +4,32 @@ const Joi = require('joi');
 
 const internals = {};
 
+// internals.model = Joi.func().keys({
+//     tableName: Joi.string().required()
+// }).unknown();
 internals.model = Joi.func();
 
 exports.plugin = Joi.object({
+
+    knexFile: Joi.object(),
+    migration: Joi.object({
```

Comment:
I do think that schwifty may have a place to make running migrations on server init, but in that case it would also be settable on a per-plugin basis (I think), probably via `server.schwifty(opts)` like other plugin-specific options.